crates/transaction-pool/src/maintain.rs

```diff
@@ -571,70 +573,76 @@ fn changed_accounts_iter(
 
 async fn apply_local_txs_backup<P>(
```

Comment:
this needs docs and I'd like to rename this to something more appropriate, perhaps

`load_and_reinsert_transactions`